teachers_digital_platform/jinja2/teachers_digital_platform/prototypes/crt-survey.html

```diff
@@ -1,148 +1,339 @@
+        </div>
+        <div class="o-survey">
+            <div class="o-survey_number">
+                <div class="h3">1.1</div>
+            </div>
+            <div class="o-survey_indicator">
+                <div class="h3">Indicator</div>
                 <p>People save for the future, and might have different goals for saving and make different choices about how to save.</p>
-                <h4>Component</h4>
-                <p>One can save money in various ways, including at home in a piggy bank or at a commercial bank, credit union, or savings and loan institution.</p>
-                <p>Saving helps individuals act on future opportunities, meet short-term and long-term goals, and address financial emergencies.</p>
-                <div class="m-form-field m-form-field__radio">
-                    <input class="a-radio" type="radio" name="radio_1" id="test_1">
-                    <label class="a-label" for="test_1">Yes</label>
+            </div>
+            <div class="o-survey_components">
+                <div class="h3">Component</div>
+                <div class="o-survey_component">
+                    <div class="o-survey_question">
+                        <p>One can save money in various ways, incl home in a piggy bank or at a commercial union, or savings and loan institution.</p>
+                    </div>
+                    <div class="o-survey_answer">
+                        <div class="m-form-field
+                                    m-form-field__radio
+                                    m-form-field__lg-target">
+                            <input class="a-radio" id="crt-question_1a" type="radio" name="crt-question-1" value="1">
+                            <label class="a-label" for="crt-question_1a">
+                                Yes
+                            </label>
+                        </div>
+                        <div class="m-form-field
+                                    m-form-field__radio
+                                    m-form-field__lg-target">
+                            <input class="a-radio" id="crt-question_1b" type="radio" name="crt-question-1" value="0">
+                            <label class="a-label" for="crt-question_1b">
+                                No
+                            </label>
+                        </div>
+                    </div>
                 </div>
-                <div class="m-form-field m-form-field__radio">
-                    <input class="a-radio" type="radio" name="radio_1" id="test_2">
-                    <label class="a-label" for="test_2">No</label>
+                <div class="o-survey_component">
+                    <div class="o-survey_question">
+                        <p>Saving helps individuals act on future opportunities, meet short-term and long-term goals, and address financial emergencies.</p>
+                        <p class="o-survey_question-helper"><small><em>Beneficial, but not essential.</em></small></p>
```

Comment:
I recommend moving the styles provided by `small` and `em` into the styles for `.o-survey_question-helper`. I don't think `small` and `em` provide useful semantic meaning here.

Reply:
Fixed